dns/zone-configs/k8s.io.yaml

```diff
@@ -240,3 +240,7 @@ velodrome:
 kind.sigs:
   type: CNAME
   value: k8s-kind.netlify.com.
+# https://github.com/kubernetes-sigs/cluster-api docs (@dwat, @jdetiber, @@justinsb)
+cluster-api.sigs:
+  type: CNAME
+  value: k8s-cluster-api.netlify.com.
```

Comment:
Would `kubernetes-sigs-cluster-api.netlify.com` be an okay alternative? That should align with the project name to be created in netlify.

Reply:
Done.